src/app/zap-templates/zcl/data-model/chip/application-basic-cluster.xml

```diff
@@ -24,13 +24,13 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>This cluster provides information about an application running on a TV or media player device which is represented as an endpoint.</description>
-    <attribute side="server" code="0x0000" define="APPLICATION_VENDOR_NAME" type="CHAR_STRING" length="32" writable="false" optional="false">vendor name</attribute>
-    <attribute side="server" code="0x0001" define="APPLICATION_VENDOR_ID" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">vendor id</attribute>
-    <attribute side="server" code="0x0002" define="APPLICATION_NAME" type="CHAR_STRING" length="32" writable="false" optional="false">application name</attribute>
-    <attribute side="server" code="0x0003" define="APPLICATION_PRODUCT_ID" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">product id</attribute>
-    <attribute side="server" code="0x0005" define="APPLICATION_ID" type="CHAR_STRING" length="32" writable="false" optional="false">application id</attribute>
-    <attribute side="server" code="0x0006" define="CATALOG_VENDOR_ID" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">catalog vendor id</attribute>
-    <attribute side="server" code="0x0007" define="APPLICATION_STATUS" type="ENUM8" min="0x00" max="0xFF" writable="false" default="0x01" optional="false">application satus</attribute>
+    <attribute side="server" code="0x0000" define="APPLICATION_VENDOR_NAME" type="CHAR_STRING" length="32" optional="false">vendor name</attribute>
+    <attribute side="server" code="0x0001" define="APPLICATION_VENDOR_ID" type="INT16U" min="0x0000" max="0xFFFF" optional="false">vendor id</attribute>
+    <attribute side="server" code="0x0002" define="APPLICATION_NAME" type="CHAR_STRING" length="32" optional="false">application name</attribute>
+    <attribute side="server" code="0x0003" define="APPLICATION_PRODUCT_ID" type="INT16U" min="0x0000" max="0xFFFF" optional="false">product id</attribute>
+    <attribute side="server" code="0x0005" define="APPLICATION_ID" type="CHAR_STRING" length="32" optional="false">application id</attribute>
+    <attribute side="server" code="0x0006" define="CATALOG_VENDOR_ID" type="INT16U" min="0x0000" max="0xFFFF" optional="false">catalog vendor id</attribute>
+    <attribute side="server" code="0x0007" define="APPLICATION_STATUS" type="ENUM8" min="0x00" max="0xFF" default="0x01" optional="false">application status</attribute>
```

Comment:
Why the removal of `writable="false"`?  These attributes are all non-writable in the spec...

Reply:
Reverted back to use writable=false. 

As we don't log the value of written attribute I thought it was due to this flag. Anyhow, I'm not able to read the value of written attribute and assert the value in tests.

Reply:
For these specific, non-writable, attributes, right?